pkgs/applications/misc/synapse/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, intltool, pkgconfig, 
+  glib, libnotify, gtk3, libgee,
+  keybinder3, json_glib, zeitgeist, vala
+}:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "synapse-0.2.99.1";
+
+  src = fetchurl {
+    url = "https://launchpad.net/synapse-project/0.3/0.2.99.1/+download/${name}.tar.xz";
+    sha256 = "846d8a5130580bb47c754bb7f20dc76311e589c00a18b02370a5d78b52409220";
+  };
+
+  buildInputs = [
+    intltool pkgconfig glib 
+    libnotify gtk3 libgee keybinder3
+    json_glib zeitgeist vala
+  ];
+
+  meta = { 
+      description = "Synapse is a semantic launcher written in Vala that you can use to start applications as
```

Comment:
Please omit pkg name in the description and make it shorter.

Add `longDescription` with more information.